subprojects/performance/src/templates/generateLotsOfDeprecationWarnings/build.gradle

```diff
@@ -17,13 +16,21 @@
 
 @groovy.transform.CompileStatic
 void createDeprecations(int iterations) {
-    for (int i=0; i<iterations; i++) {
+    for (int i = 0; i < iterations; i++) {
         // every 10th deprecation should have a different message
-        if(i%10 == 0) {
-            org.gradle.util.DeprecationLogger.nagUserOfDeprecated("Some unique deprecation No #\$i")
```

Comment:
Can you as well delete `org.gradle.util.DeprecationLogger` from our code base?

Reply:
Shall we remove it immediately or wait for 7.0? I'm not sure how community members use this API, but myself uses it in my opensource plugin...

Reply:
Hmmm...

It is an internal API, so we could change it however we want. Do we have any failing smoke tests when we remove the old deprecation API? Do we need to add a deprecation warning when someone uses the old API?